content/news/playtest-20160424.md

```diff
@@ -0,0 +1,77 @@
+* Your OS version (e.g. "Windows: Microsoft Windows NT 6.1.7601 Service Pack 1" or "OSX: Unix 14.5.0.0")
+* Your .NET runtime version (e.g. ".NET CLR 4.0.30319.42000" or "Mono 4.0.2 ((detached/c99aa0c) CLR 4.0.30319.17020")
+* Your GL driver version (e.g. "4.5.0 NVIDIA 352.63" or "3.0 Mesa 10.5.9")
+* Your OS language setting (e.g. "en" or "ru")
+* The OpenRA version / mod / modversion that you launched
+
+<div style="text-align:center" markdown="1">
+![Share of operating systems of our users](/images/news/20160424-os-shares.png)
+This is the share of operating systems among those players who used the playtest.
+</div>
+
+<hr />
+
+###Changes to map handling
+
+Custom maps are now stored in an additional sub-directory per mod version (e.g. `My Documents\OpenRA\maps\ra\playtest-20160403` on Windows).
+
+To ensure that outdated maps will not cause problems during gameplay, these maps will not be automatically moved when you upgrade the game. You will have to download updated versions from the [Resource Center](http://resource.openra.net) instead, or upgrade and move them yourself (see below). Note that the maps from the first playtest are compatible with this one (and will very likely also be compatible with the final release), so you can just copy them over, or rename the directory. 
+
+###Map format changes
+
+The current map format (from playtest-20160403 onwards) is version 11.
+
+In previous versions (up to release-20151224), the game used to automatically update maps with earlier map formats to the latest one supported by that version. As of playtest-20160403, this is no longer the case. To update maps, you have three options:
+
+1. Use the `--upgrade-map` command of the OpenRA.Utility.
+2. Download an updated version of the map from the Resource Center, or
+3. Use the Resource Center to upgrade the map.
+
+<div style="text-align:center" markdown="1">
+![Upgrading maps in the resource center](/images/news/20160403-web-resource-center-map-upgrade.png)
+</div>
+
+The new map format brings with it a couple advantages:
+
+* Minimap preview is embeddeded into the map package as a separate PNG file,<br/>allowing third-party tools and websites immediate access.
+* YAML rules and Lua scripts can be split into multiple files, which can be shared among multiple maps.
+
+One noteable change that needs explanation is the removal of the `Description:`, `Options:`, and `Videos:` blocks, whose properties have been moved to their corresponding traits. During map upgrades, these translations will happen automatically. Take a look at [one of our single-player missions](https://github.com/OpenRA/OpenRA/blob/playtest-20160403/mods/ra/maps/allies-01/rules.yaml) and the [Fort LoneStar map](https://github.com/OpenRA/OpenRA/blob/playtest-20160403/mods/ra/maps/fort-lonestar/rules.yaml) to see how these work now, or search for the following traits in our [trait documentation](http://wiki.openra.net/Traits%20(Playtest)):
```

Comment:
I'm quite sure `Description` is gone entirely (wasn't moved to a trait, rather the resource center).